client-side/src/staticUser.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+
+const staticUser = {
+    "_id": "669f86c302f253ec5a6f5162",
+    "name": "Test-User",
+    "email": "testuser@example.com",
+    "password": "$2b$10$IlUR0EDO.Eu8O4K8BgaFfeXWgeLxdZ6WSw0GXBWm7XZDZ1ouQmdeC",
+    "formatedDate": "yyyy-MM-dd",
+    "profileImage": "profile.jpg",
+    "visitsWebsites": [],
+    "profiles": [],
+    "preferences": [{
+        "emailFrequency": "never",
+        "sendNotificationTime": 20,
+        "soundVoice": "mmm.mp3",
+        "_id": "66a0b3c52dd0dde7df269f6a",
+        "__v": 0
+    }]
+};
+
+const UserProfile = () => {
+  return (
+    <div>
+      <h2>User Profile</h2>
+      <img src={staticUser.profileImage} alt={`${staticUser.name}'s profile`} />
+      <p><strong>Name:</strong> {staticUser.name}</p>
```

Comment:
please don't use p and strong - those are simple HTML elements.
also, what is this component used for?